### PR TITLE
INREL-3967: Merge infinite amp theme 8.x-2.x-dev into 8.x-2.x

### PR DESCRIPTION
**Pull request by alxshll**
## [INREL-3967](https://jira.burda.com/browse/INREL-3967) 
Merge 8.x-2.x-dev into 8.x-2.x

## How to test:
Please check to see if anyone can foresee any issues with this merge

### CODE REVIEW

**Comment by mgoedecke** [comment]
lgtm

**Review by mgoedecke — APPROVED** [review]
lgtm

File: templates/system/page--front.html.twig [hunk 1]
```diff
@@ -0,0 +1,67 @@
+{#
```
Comment (saviomuc) [inline templates/system/page--front.html.twig]: Needs to be moved to Instyle

File: templates/paragraphs/paragraph--social.html.twig [hunk 2]
```diff
@@ -52,4 +52,7 @@
     {% else %}
         {{ content.field_link }}
     {% endif %}
-{% endblock %}
\ No newline at end of file
+
+    {{ content.field_embed_description }}
```
Comment (saviomuc) [inline templates/paragraphs/paragraph--social.html.twig]: Needs to be moved to instyle

File: templates/field/field--paragraph--field-embed-description.html.twig [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+ * - field_name: The name of the field.
+ * - field_type: The type of the field.
+ * - label_display: The display settings for the label.
+ *
+ * @see field.html.twig
+ *
+ * @ingroup themeable
+ */
+#}
+  {%- for item in items -%}
```
Comment (saviomuc) [inline templates/field/field--paragraph--field-embed-description.html.twig]: Needs to be moved to Instyle

**Review by saviomuc — CHANGES_REQUESTED** [review]

**Comment by alxshll** [comment]
@saviomuc Everything done. See also here: https://github.com/BurdaMagazinOrg/instyle-web/pull/360/commits

**Review by saviomuc — APPROVED** [review]
LGTM :shipit:

**Pull request merged**